concepticondata/conceptlists/Maciejewski-2016-100.tsv

```diff
@@ -0,0 +1,201 @@
+Maciejewski-2016-100-97	97	peel	1	to pull, tear, or cut the outer covering from	PEEL	275	90.41	0.91	91	7.79	0.89	2.79	7	6	1.46	4.75	4.32	4.23
+Maciejewski-2016-100-98	98	peel	2	a spadelike instrument with a long handle used for putting goods to be baked into an oven, or for removing them again			8.64	0.91	91	7.79	0.89	2.79	7	6	1.46	4.75	4.32	4.23
+Maciejewski-2016-100-99	99	peer	1	a person of the same rank, status, age group, ability, or the like as another person			55.28	0.37	65	9.36	0.97	3	4	3	1.81	3.11	4.06	4.85
+Maciejewski-2016-100-100	100	peer	2	to look closely, searchingly, or with difficulty			44.72	0.37	65	9.36	0.97	3	4	3	1.81	3.11	4.06	4.85
+Maciejewski-2016-100-101	101	pen	1	any of various instruments used for writing or drawing in ink	PEN	794	74.58	0.64	78	20	1.3	2.8	7	6	1.76	5.76	5.71	5.54
+Maciejewski-2016-100-102	102	pen	2	a small, usually fenced enclosure for animals			25.12	0.64	78	20	1.3	2.8	7	6	1.76	5.76	5.71	5.54
+Maciejewski-2016-100-103	103	pet	1	an animal, usually domesticated or tamed, kept in one's home for companionship rather than practical use			91.2	0.9	93	14.5	1.16	3.02	8	7	1.39	5.89	5.57	5.41
+Maciejewski-2016-100-104	104	pet	2	a sulky mood or fit of temper; peeve; petulance			8.4	0.9	93	14.5	1.16	3.02	8	7	1.39	5.89	5.57	5.41
+Maciejewski-2016-100-105	105	pine	1	any of numerous evergreen trees that have clustered, needle-shaped leaves and bear cones	PINE	1740	63.47	0.46	76	11.01	1.04	3.03	4	3	1.47	4.78	5.92	4.62
+Maciejewski-2016-100-106	106	pine	2	to be affected with great desire and longing (often followed by for)			36.53	0.46	76	11.01	1.04	3.03	4	3	1.47	4.78	5.92	4.62
+Maciejewski-2016-100-107	107	pink	1	a pale reddish color; the color that results when white and red paint are mixed	PINK	3446	93.85	0.95	98	33.22	1.52	2.8	8	7	1.54	NA	NA	NA
+Maciejewski-2016-100-108	108	pink	2	to cut a scalloped or notched edge on			5.8	0.95	98	33.22	1.52	2.8	8	7	1.54	NA	NA	NA
+Maciejewski-2016-100-109	109	pit	1	a comparatively wide and deep hole dug or existing in the ground	PIT (POTHOLE)	2539	71.82	0.63	59	17.47	1.24	2.27	16	11	1.67	5.89	5.93	4.93
+Maciejewski-2016-100-110	110	pit	2	the hard seed at the center of an apricot, cherry, plum, or certain other fruits			24	0.63	59	17.47	1.24	2.27	16	11	1.67	5.89	5.93	4.93
+Maciejewski-2016-100-111	111	plane	1	a flat or level surface			33.99	0.52	78	36.2	1.56	3.03	10	9	1.58	5.56	5.35	5.58
+Maciejewski-2016-100-112	112	plane	2	any of various carpentry tools with an adjustable blade, used to make surfaces of wood smooth, true, grooved, or the like	PLANE (TOOL)	3408	26.29	0.52	78	36.2	1.56	3.03	10	9	1.58	5.56	5.35	5.58
+Maciejewski-2016-100-113	113	plot	1	the story line or sequence of events in a novel, play, or the like	PLOT	2024	57.66	0.36	60	22.07	1.34	2.53	8	7	1.67	4.2	3.79	5.15
```

Comment:
PLOT in concepticon refers rather to a sort of conspiracy (also according to the glosses mapped to it).